package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "cht-conf-publish-test",
```

Comment:
Not related but good catch.

Reply:
I think that was fixed when I did the `npm version` call! I didn't even notice the change....